packages/mobile/src/components/image/CollectionImage2.tsx

```diff
@@ -0,0 +1,102 @@
+import type { SquareSizes, ID } from '@audius/common/models'
+import {
+  cacheCollectionsSelectors,
+  reachabilitySelectors
+} from '@audius/common/store'
+import type { Maybe } from '@audius/common/utils'
+import { useSelector } from 'react-redux'
+
+import { FastImage, useTheme } from '@audius/harmony-native'
+import type { FastImageProps } from '@audius/harmony-native'
+import imageEmpty from 'app/assets/images/imageBlank2x.png'
+import { useContentNodeImage } from 'app/hooks/useContentNodeImage'
+import { getLocalCollectionCoverArtPath } from 'app/services/offline-downloader'
+import { getCollectionDownloadStatus } from 'app/store/offline-downloads/selectors'
+import { OfflineDownloadStatus } from 'app/store/offline-downloads/slice'
+
+import { primitiveToImageSource } from './primitiveToImageSource'
+
+const { getIsReachable } = reachabilitySelectors
+const { getCollection } = cacheCollectionsSelectors
+
+const useLocalCollectionImageUri = (collectionId: Maybe<ID>) => {
```

Comment:
nit: is `Maybe` not just the same as `collectionId?: ID`?

Reply:
good catch!